zh/deploy-tidb-binlog.md

```diff
@@ -386,6 +396,10 @@ spec:
 
 ### 完全移除 Pump 节点
 
+> **注意：**
+>
+> 完全移除 Pump 节点前，需要至少存在一个 Pump 节点。如果此时 Pump 节点已经缩容到 0，需要先至少扩容到 1，再进行下面的移除操作。
```

Comment:
Is there a doc for scaling out Pump nodes that we can link to?

Reply:
There's no individual Pump scale-out docs, because Pump itself is in the process of deprecation.
Instructions about scaling out Pump is added in-place.